util/steambox

```diff
@@ -51,9 +51,14 @@ declare -a DRIDEVS=()
 if [[ -d /dev/dri ]] ; then
 	DRIDEVS+=(/dev/dri/*)
 fi
-if [[ -d /dev/ati ]] ; then 
+if [[ -d /dev/ati ]] ; then
 	DRIDEVS+=(/dev/ati/*)
 fi
+for cdev in nvidiactl nvidia0 nvidia-modeset; do
```

Comment:
@jcorrado My one thought on this: if someone is running two graphics cards (for example, an SLI build), will they possibly have `/dev/nvidia1` as well?

The possibility definitely won't stop me from merging this, but I wonder if we should say `nvidia[0-9]`